collector/helpers.go

```diff
@@ -0,0 +1,20 @@
+package collector
+
+import (
+	"github.com/google/gnxi/utils/xpath"
+	"github.com/openconfig/gnmi/proto/gnmi"
+)
+
+func CreatePrefix(prefix, target string) (*gnmi.Path, error) {
+	if len(prefix)+len(target) == 0 {
+		return nil, nil
+	}
+	p, err := xpath.ToGNMIPath(prefix)
```

Comment:
shouldn't we use [collector.ParsePath](https://github.com/karimra/gnmic/pull/84/files#diff-bfea94335b13a4db753f76deaabe014bR12) here to allow setting the origin to the prefix as well?

Reply:
- Yes but that's part of another branch, once it is merged this should change